src/js/view/floating/floating-controller.ts

```diff
@@ -171,6 +172,11 @@ export default class FloatingController {
                     });
                 });
             }
+            setTimeout(() => {
+                this.transitionFloating(false);
+                // Perform resize and trigger "float" event responsively to prevent layout thrashing
+                this._model.trigger('forceResponsiveListener', {});
+            }, 50);
```

Comment:
The timeout here is arbitrary. It could be tuned but any listening for changes would likely only force layout. The actual CLS we're getting is often caused by the animations we're using (depended on which player element caused a change (controls, overlay, parent container because of inner changes...)